coopy/DiffRender.hx

```diff
@@ -359,8 +365,11 @@ class DiffRender {
                             corner,
                             cell,
                             off);
-                render.insertCell(pretty_arrows?cell.pretty_value:cell.value,
-                                  cell.category_given_tr);
+                var val = pretty_arrows?cell.pretty_value:cell.value;
+                if (quote_html) {
+                    val = StringTools.htmlEscape(view.toString(val));
```

Comment:
@paulfitz Wouldn't it also make sense to pass [`quotes=True`]( https://api.haxe.org/StringTools.html#htmlEscape) by default here?

Reply:
I'm not against it, but I don't see the value of doing so - the escaped material is not placed anywhere where quotation marks would be significant?  Do you see a concrete problem without quotes=True?

Maybe I chose a misleading name for the variable...

Reply:
You're right. That is not needed here. Sorry about that.